book/pipeline-service.md

```diff
@@ -1,29 +1,27 @@
 # Pipeline Service
 
-Pipeline Service provides Tekton APIs and services to kcp workspaces.
-When a workspace binds to the Pipeline Service APIExport, it has access to the following Tekton APIs:
+Pipeline Service provides a SaaS for pipelines. It leverages:
 
-- `Task`
-- `Pipeline`
-- `PipelineRun`
-- `Repository` (for [Pipelines as Code](https://pipelinesascode.com))
+- Kubernetes / OpenShift for the compute
+- Tekton Pipelines, Results and friends for the core of the service
+- OpenShift GitOps / Argo CD, Pipelines as Code for managing the infrastructure
 
-The following features are also provided:
+The following features are provided:
 
 - Signing and attestation of `TaskRuns` with [Tekton Chains](https://tekton.dev/docs/chains/)
-- Archiving of `PipelineRuns` and `TaskRuns` with [Tekton Results](https://tekton.dev/docs/results/)
+- Archiving/Pruning of `PipelineRuns` and `TaskRuns` with [Tekton Results](https://tekton.dev/docs/results/)
 - Integrations with GitHub, Gitlab, and Bitbucket with [Pipelines as Code](https://pipelinesascode.com) (PaC).
 
 ## Architecture
 
-Tekton controllers are deployed directly on the workload cluster.
-The kcp syncer for the workload cluster is configured to sync `Pipeline`, `Task`, and `PipelineRun` objects.
-When a `PipelineRun` is created on kcp, it is synced to the workload cluster whose controllers then create `TaskRun` objects and resulting `Pods`.
-The Chains and Results controllers then work with the synced `PipelineRun` and generated `TaskRun` objects to perform their respective actions.
+Pipeline Service is deployed onto the cluster alongside other AppStudio components. The following form the core architecture of Pipeline Service:
 
-Like upstream Tekton, Pipelines as Code (PaC) also runs controllers directly on workload clusters.
-Unlike other Tekton components, PaC also exposes a `Route`/`Ingress` that allows it to receive webhook events from source control repositories.
-To sent traffic from kcp to the workload cluster, Pipeline Service will deploy a "gateway" service that acts as a proxy to PaC on the workload cluster.
+1. _OpenShift Pipelines:_ It enables the creation and management of Pipelines for user Applications.
+2. _Pipelines as Code (PaC):_ An OpenShift Pipelines component that enables receiving webhook events from version control systems. User would install a Github App to configure this.
```

Comment:
I propose to say "User would have to install a [Github App](https://pipelinesascode.com/docs/install/github_apps/) having the proper rights to forward events to the PaC endpoint exposed by the platform.

Reply:
Done.